src/firefly/js/templates/router/RouteHelper.jsx

```diff
@@ -114,7 +119,6 @@ export function useDropdownRoute() {
             const menuItem = getMenuItem(pathname);
             if (menuItem) {     // the requested pathname is in the menu
                 dispatchSetLayoutInfo({dropDown:{view: menuItem.action, menuItem, visible: true}});
-                dispatchSetMenu({selected: menuItem.action});
```

Comment:
@loitly I was able to fix this by just getting rid of this line. `dispatchSetLayoutInfo` in the line above seems to be enough to make this work.

Reply:
You are right.  That line should be enough.  Basically, you want to switch to that menu tab/item.